doc/source/user/basics.indexing.rst

```diff
@@ -424,7 +743,7 @@ supplies to the index a tuple, the tuple will be interpreted
 as a list of indices. For example (using the previous definition
 for the array z): ::
 
- >>> indices = (1,1,1,1)
+ >>> indices = (1, 1, 1, 1)
```

Comment:
This references an array `z` that no longer exists (or at least is very far away from the current example).
```suggestion
 >>> z = np.arange(81).reshape(3, 3, 3, 3)
 >>> indices = (1, 1, 1, 1)
```
Note also that the text above the example should be modified to reflect this change.